setup.cfg

```diff
@@ -42,6 +42,7 @@ dev =
   python-dotenv==1.0.0
   ruff==0.0.292
   pytest-asyncio==0.23.5
+  openai==1.12.0
```

Comment:
Can we please remove the strict version check to avoid conflicts.

Reply:
Done